### PR TITLE
Or 133 fixes

### PR DESCRIPTION
**Pull request by ankitShogun**
Ticket - https://fide-official.atlassian.net/browse/OR-133

### CODE REVIEW

**Review by nikhil2128 — APPROVED** [review]
Lots of commented code. Remove in your next PR

File: components/header/language/Language.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ const Language = () => {
   const dispatch = useDispatch();
   const [openLang, setOpenLang] = useState(false);
 
-  useEffect(() => {
-    document.documentElement.dir = locale === "en" ? "ltr" : "rtl";
-  }, [locale]);
+  // useEffect(() => {
```
Comment (ujjwal502) [inline components/header/language/Language.tsx:13]: do we need this later?

File: pages/index.tsx [hunk 2]
```diff
@@ -111,7 +114,9 @@ const Homepage = () => {
 		tagValue: string,
 		tagName: string
 	) => {
-		let url = `${process.env.NEXT_PUBLIC_BECKN_API_URL}/stores?tagName=${tagName}&tagValue=${tagValue}&latitude=${lat}&longitude=${long}`;
+		// let url = `${process.env.NEXT_PUBLIC_BECKN_API_URL}/stores?tagName=${tagName}&tagValue=${tagValue}&latitude=${lat}&longitude=${long}`;
```
Comment (ujjwal502) [inline pages/index.tsx:117]: same comment as above?

File: pages/orderDetails.tsx [hunk 3]
```diff
@@ -123,8 +123,8 @@ const OrderDetails = () => {
 
   return (
     <>
-      <AppHeader appHeaderText={t.orderDetails} />
-      <Accordion accordionHeader={t.order}>
+      {/* <AppHeader appHeaderText={t.selectPaymentMethod} /> */}
```
Comment (ujjwal502) [inline pages/orderDetails.tsx:126]: Can we remove this?

File: public/sitemap-0.xml
Comment (ujjwal502) [inline public/sitemap-0.xml:1]: Are we pushing build outputs?

**Pull request merged**